docs/modules/develop/pages/backports.adoc

```diff
@@ -0,0 +1,14 @@
+= Backports
+
+We only offer support for the last two mayor version.
+
+For instance, if we're on the version `0.28.0.dev`, we make backports for `v0.26.x` and `v0.27.x`. We only backport fixes and not features, as we want to respect xref:develop:guide_conventions.adoc[Semantic Versioning].
+
+The idea of a backport is to bring a given fix from the last development version to an older release.
+
+The process for making backports is the following:
+
+. Check in the git commit history what was the latest backports on the branch that you're working with. https://github.com/decidim/decidim/commits/release/0.27-stable[Example for v0.27]. Copy the number of the last PR backported.
+. Search inn GitHub for the this PR in the list of merged PRs of the `type: fix`. https://github.com/decidim/decidim/pulls?page=1&q=is%3Apr+sort%3Aupdated-desc+label%3A%22type%3A+fix%22+is%3Amerged[Example URL].
```

Comment:
Remove a "n" from Search in**n** Github

Reply:
Fixes in 7f8ef535efad947fc86f6851f75fa71a47a8864a 
For this kind of reviews I'm a fan of the "add a suggestion" feature in GitHub 😄

Reply:
That is great! 
When i have reviewed the PR, i could not find the functionality (did not knew where the button is). Now i do!